libkineto/src/RoctracerActivityApi.cpp

```diff
@@ -75,6 +80,8 @@ int RoctracerActivityApi::processActivities(
   clock_gettime(CLOCK_REALTIME, &t00);
 
   const timestamp_t toffset = (timespec_to_ns(t0) >> 1) + (timespec_to_ns(t00) >> 1) - timespec_to_ns(t1);
+  startTime = (startTime * 1000) - toffset;
+  endTime = (startTime * 1000) - toffset;
```

Comment:
should this be endTime? 
```
endTime = (startTime * 1000) - toffset;
```

Reply:
Good eye, thanks. Did a copy/paste back out of kineto@pytorch_1.13 branch I was testing in.  Update momentarily.